assets/package.json

```diff
@@ -75,11 +75,11 @@
     "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-replace": "^2.2.0",
     "rollup-plugin-uglify": "^6.0.3",
-    "uglify-js": "^3.6.0",
-    "uglifycss": "0.0.29",
-    "wdfn-viz": "1.0.0"
+    "stylelint": "^10.1.0",
+    "stylelint-scss": "^3.11.1"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-free": "5.11.2",
```

Comment:
Moved this dependency and wdfn-viz from dev-dependencies. Not sure why we put them there. The are really application dependencies and all of the dependencies have to be loaded in order to build.